pkgs/development/tools/glamoroustoolkit/default.nix

```
@@ -0,0 +1,95 @@
+{ stdenv, lib, makeWrapper, fetchzip
+, gtk3
+, glib
+, libX11
+, cairo
+, libglvnd
+, fontconfig
+, freetype
+, libxcb
+, libXrender
+, libXext
+, libuuid
+, dbus
+, libXcursor
+, libXrandr
+, libXi }:
+stdenv.mkDerivation rec {
+  version = "0.7.2";
+  name = "GlamorousToolkitVM-${version}";
+
+  src = fetchzip {
+    url = "https://github.com/feenkcom/gtoolkit-vm/releases/download/v0.7.2/GlamorousToolkit-x86_64-unknown-linux-gnu.zip";
+    sha256 = "Tahe/P0sxmuhSG5JxOkyQGLTOyVrjMEkxvzPc2CBV0Y=";
+    stripRoot = false;
+  };
+  sourceRoot = ".";
+
+  dontConfigure = true;
+  dontBuild = true;
+  dontPatchELF = true;
+
+  installPhase = ''
+    mkdir $out
+    cp -a $src/bin $src/lib $out/
+    chmod +w $out/bin
+    cp ${./GlamorousToolkit-GetImage} $out/bin/GlamorousToolkit-GetImage
+    chmod -w $out/bin
```

This is strange. Have you tried to create the directory tree first?

```suggestion
    install -d $out/bin $out/lib
    cp -r $src/bin $src/lib $out/
    cp ${./GlamorousToolkit-GetImage} $out/bin/GlamorousToolkit-GetImage
```